linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/dao/impl/ConfigMapper.xml

```diff
@@ -92,6 +92,11 @@
         v.id as value_id, v.config_key_id,config_value, v.config_label_id
     </sql>
 
+    <sql id="Config_En_Key_Value_Column">
+        k.id, k.key, k.en_description as description, k.en_name as name, k.engine_conn_type, k.default_value, k.validate_type, k.validate_range, k.is_hidden, k.is_advanced, k.level, k.en_treeName as treeName,
+        v.id as value_id, v.config_key_id,config_value, v.config_label_id
```

Comment:
Try to control no more than three fields per line

Reply:
done